.github/workflows/ci_web.yml

```diff
@@ -25,20 +9,12 @@ jobs:
       run:
         working-directory: web
     steps:
+      - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version: lts/*
-      - uses: actions/checkout@v4
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/cache@v4
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          cache: yarn
```

Comment:
Question ✋ 

Does this cache strategy work?

`actions/setup-go` (the only action I know) stores the caches **per branch**,  if this `setup-node` action has the same strategy (likely be same), then it won't work as expected.

Reply:
AFAIK the default branch cache is shared with other branches, and another useful use case is when multiple pushes to the same branch happen then the cache can be useful

<img width="818" alt="image" src="https://github.com/user-attachments/assets/ae72f2c1-8c2c-4999-a3d3-cc930b3012b9">